rust/Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
@calbaker do you know what this does? I remember seeing obnoxious messages about it when compiling.

Reply:
@kylecarow , it solves obnoxious messages!